package.json

```diff
@@ -12,6 +12,7 @@
 		"lint:js": "xo",
 		"pack:safari": "xcodebuild -project 'safari/Refined GitHub.xcodeproj' -scheme 'Refined GitHub (macOS)'",
 		"start:safari": "open 'safari/build/Release/Refined GitHub.app'",
```

Comment:
This command is also broken since the iOS PR probably, we should drop it if there's no fix
```suggestion
```

Reply:
Yeah, it's better to just build and run in Xcode.

Reply:
The point is that I try to avoid opening XCode at all because I shouldn't have to deal with it. That's why there's a build:safari script as well